src/explore-education-statistics-frontend/src/modules/methodologies/components/MethodologyList.tsx

```diff
@@ -16,26 +16,17 @@ function MethodologyList({ publications }: Props) {
   }
 
   return (
-    <ul>
+    <ul data-testid="methodology-list">
       {filteredPublications.map(publication => (
-        <li className="govuk-!-margin-bottom-2" key={publication.id}>
-          <h3
-            className="govuk-heading-s govuk-!-margin-bottom-0"
-            id={`methodology-heading-${publication.id}`}
-          >
-            {publication.title}
-          </h3>
-
-          <ul>
-            {publication.methodologies.map(methodology => (
-              <li key={methodology.id}>
-                <Link to={`/methodology/${methodology.slug}`}>
-                  {methodology.title}
-                </Link>
-              </li>
-            ))}
-          </ul>
-        </li>
+        <>
+          {publication.methodologies.map(methodology => (
+            <li key={methodology.id} className="govuk-!-margin-bottom-2">
+              <Link to={`/methodology/${methodology.slug}`}>
+                {methodology.title}
+              </Link>
+            </li>
+          ))}
+        </>
```

Comment:
Code-wise this is fine and is what the ticket said to do, but I think it will cause some different issues.

The problem is that we're listing the methodologies by publication, and methodologies often have the same name as the publication, so it looks like the methodology name is being duplicated. However, methodology names can be different to the publication and they can be shared among multiple publications, so by removing the heading we're removing the context and in some cases will end up with a list of duplicate links. For example Pupil Absence has 3 publications that share 'Pupil absence statistics: methodology', so without the publication titles we'd just have 3 links to the same methodology. 

<img width="543" alt="methodologylist" src="https://user-images.githubusercontent.com/81572860/190370701-1bb1a54e-793f-4de1-a057-61961c950735.PNG">

I'm not sure what the best solution for this is, probably a question for @mrmarvino